src/senaite/core/upgrade/v02_03_000.py

```diff
@@ -420,3 +426,49 @@ def fixed_point_value_to_string(value, precision):
     str_value = template % (sign, front, fra)
     # strip off trailing zeros and possible dot
     return str_value.rstrip("0").rstrip(".")
+
+
+def setup_catalogs_order(portal):
```

Comment:
Would you mind to add this as a setuphandler step and use it from there?

Reply:
Sure

Reply:
Done with https://github.com/senaite/senaite.core/pull/2146/commits/70b363d487e34653e137c3013f7ced38cc822cc2